routes/display/Display.js

```diff
@@ -1,15 +1,14 @@
 // All routes related to club's LOGIN AND REGISTER
 
 const express = require("express");
-const Club = require("../../schema/club/ClubSchema");
 const User = require("../../schema/user/UserSchema");
 const { STATUSCODE } = require("../../utils/Status");
 const router = express.Router();
 
 // Route 1  - Show all avaialble Users in the DB
 router.get("/users", async (req, res) => {
   try {
-    const allusers = await User.find({});
+    const allusers = await User.find({ usertype: "individual" });
```

Comment:
In display js 2 routers are there 1 for all clubs and 1 for all Users. So I thought it would have been better if we use both for different usertypes.  Anyway I changed it back.

![image](https://github.com/MilanCommunity/Milan-Backend/assets/113163315/afd6eff1-5b29-4b56-a3c4-df23a1db7c06)

Reply:
The display route won't be used anymore, if you look at the front end we are displaying the clubs by /clubs  and not the /display/clubs.